src/test/ui/async-await/issue-70594.stderr

```diff
@@ -25,11 +25,12 @@ LL |     [1; ().await];
    |         ^^^^^^^^ `()` is not a future
    |
    = help: the trait `Future` is not implemented for `()`
-note: required by `poll`
-  --> $SRC_DIR/core/src/future/future.rs:LL:COL
+   = note: required because of the requirements on the impl of `IntoFuture` for `()`
+note: required by `into_future`
+  --> $SRC_DIR/core/src/future/into_future.rs:LL:COL
    |
-LL |     fn poll(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output>;
-   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+LL |     fn into_future(self) -> Self::Future;
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
These will likely conflict with / be cleaned up by #90939

Reply:
Thanks for the heads up. I just resolved the conflict, and I think the error message does look much better.